docs/reference/edot-collector/components/elasticapmprocessor.md

```diff
@@ -0,0 +1,146 @@
+| Service metadata | Ensures service name, version, and environment are correctly formatted. |
+| Span metadata | Enriches spans with Elastic-specific fields for proper rendering in trace views. |
+| Error handling | Transforms error information to align with Elastic's error model. |
+| Data stream routing | Adds necessary attributes for proper data stream routing in {{es}}. |
+
+For detailed information about specific attributes and transformations, refer to the [upstream `elasticapmprocessor` documentation](https://github.com/elastic/opentelemetry-collector-components/tree/main/processor/elasticapmprocessor).
+
+## Best practices
+
+Follow these recommendations when using the Elastic APM processor:
+
+* **Always pair with the elasticapm connector**: The processor and connector work together to provide the full Elastic {{product.apm}} experience. The processor enriches traces while the connector generates {{product.apm}} metrics. Include both in your pipeline configuration for complete functionality.
+
+* **Place after batching in the pipeline**: Configure the Elastic APM processor after the batch processor to ensure optimal throughput. Batching first reduces the number of processing operations.
+
+* **Use only for direct {{es}} ingestion**: If you're using the {{motlp}}, you don't need the Elastic APM processor, because the endpoint handles enrichment automatically. Using both can cause conflicts or duplicate processing.
+
+* **Keep the processor updated**: The Elastic APM processor evolves with new Elastic {{product.apm}} features. Keep your EDOT Collector version current to benefit from the latest enhancements and compatibility improvements.
+
+* **Configure OTel SDKs with semantic conventions**: The processor relies on OpenTelemetry semantic conventions to identify and enrich trace data correctly. Ensure your SDKs follow standard conventions for service name, span attributes, and resource attributes.
+
+## Limitations
+
+Be aware of these constraints and behaviors when using the Elastic APM processor:
+
+* **Required for Elastic {{product.apm}} UIs**: Without the Elastic APM processor, OpenTelemetry traces will be stored in {{es}} but may not render correctly in Elastic {{product.apm}} UIs. Service maps, transaction groups, and other {{product.apm}}-specific visualizations depend on the enrichments this processor provides.
+
+* **Not available in upstream OTel Collector**: The Elastic APM processor is an Elastic-specific component not included in the standard OpenTelemetry Collector or Collector Contrib distributions. To use it, you must either use EDOT Collector or [build a custom collector](../custom-collector.md) that includes Elastic's components.
+
+* **Works only with OTel-native mode**: The processor is designed for use with the `elasticsearch` exporter in `otel` mode. It doesn't apply transformations for ECS-mode exports, as those follow a different data model.
```

Comment:
```suggestion
```

I'd drop this - this isn't completely true anymore. As we worked on replicating MIS's functionality, it turned out, we actually need an ecs mode as well, which the processor now supports.